doc/tutorial/conv_arithmetic.txt

```diff
@@ -0,0 +1,834 @@
+.. _conv_arithmetic:
+
+===============================
+Convolution arithmetic tutorial
+===============================
+
+.. note::
+
+    This tutorial is adapted from an existing `convolution arithmetic guide
+    <https://arxiv.org/abs/1603.07285>`_ [#]_, with an added emphasis on
+    Theano's interface.
+
```

Comment:
We *might* want to also add a disclaimer that points out that the signal processing community has a different nomenclature and a well established literature [cite: we got some references in the feedback of the paper IIRC] on the topic, but in the course of this tutorial we will stick to the terms used in the machine learning community.